release_notes/markdown.py

```diff
@@ -0,0 +1,149 @@
+import dataclasses
+import typing
+from collections import defaultdict
+
+import release_notes.model as rnm
+
+
+@dataclasses.dataclass
+class Title:
+    display: str
+    identifiers: list[str]
+
+    def __hash__(self):
+        return hash(self.display)
+
+    def __eq__(self, other):
+        return (
+            other is not None
+            and isinstance(other, Title)
+            and (other == self or hash(other) == hash(self))
+        )
+
+
+def _simple_title(display: str) -> Title:
+    return Title(display, [display.lower()])
+
+
+categories: list[Title] = [
+    Title("⚠️ Breaking Changes", ["action", "breaking"]),
```

Comment:
use single-quotes

Reply:
That'll teach me to import other people's code I suppose 😄. Will do (also for the other cases in this file)